game/scripts/npc/abilities/spirit_breaker_greater_bash.txt

```diff
@@ -38,7 +38,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "damage"                                          "10 25 40 55 120 170" // OAA
+        "damage"                                          "30 45 60 75 140 210" // OAA
```

Comment:
i think level 1 is an overnerf... what about `20 30 40 50 125 200`?

Reply:
But I reverted my nerf to 30/45/60/75/140/210.